### DISCUSSION TITLE
set the min-width of the window

### DISCUSSION THREAD

**Discussion by ariear** [tauri-apps/tauri#7525] [Q&A]
How can i set the min-width of the window?, i use windows 10 btw

**Comment by FabianLars** [comment]
there are minWidth / min_width configs/apis in js and rust and [tauri.conf.json](https://tauri.app/v1/api/config/#windowconfig.minwidth), so it depends on how/where you create the window. Also as a hint, you currently have to also set the min heigh for min width to work  (will be fixed in an upcoming release).
